proprietary/tokens/src/components/ams/dialog.tokens.json

```
@@ -2,7 +2,7 @@
   "ams": {
     "dialog": {
       "background-color": { "value": "{ams.color.background}" },
-      "border": { "value": "0" },
+      "border": { "value": "{ams.border.width.md} solid {ams.color.background}" },
```

Don't we want to use the Dialog background color here? If that ever changes, the border should get the same color. 

```suggestion
      "border": { "value": "{ams.border.width.md} solid {ams.dialog.background-color}" },
```